composer/spec/dependabot/composer/update_checker_spec.rb

```diff
@@ -437,11 +437,15 @@
       # This unit test is testing that a dependency located on https://pear.horde.org is still correctly
       # handled by composer. So ignore the fact that this package actually exists on packagist, and
       # pretend it just 404's.
-      let(:packagist_response) { '{"error":{"code":404,"message":"Not Found"}}' }
+      let(:packagist_response) { "404 not found, no packages here" }
```

Comment:
The `404` error message returned by packagist changed slightly from `v1` to `v2`... this is the correct `v2` error message.